internal/repository/root_test.go

```diff
@@ -43,6 +42,106 @@ func TestInitializeRoot(t *testing.T) {
 	assert.Nil(t, err)
 }
 
+func TestAddRootKey(t *testing.T) {
+	r, keyBytes := createTestRepositoryWithRoot(t, "")
+
+	key, err := tuf.LoadKeyFromBytes(keyBytes)
+	if err != nil {
+		t.Fatal(err)
+	}
+	sv, err := signerverifier.NewSignerVerifierFromSecureSystemsLibFormat(keyBytes)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	var newRootKey *sslibsignerverifier.SSLibKey
+
+	newRootKey, err = tuf.LoadKeyFromBytes(targetsKeyBytes)
```

Comment:
```suggestion
	newRootKey, err := tuf.LoadKeyFromBytes(targetsKeyBytes)
```

This can now remove the `newRootKey` declaration right?

Reply:
Removed